crates/holochain_json_api/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "holochain_json_api"
-version = "0.0.46"
+version = "0.0.47"
```

Comment:
There's a nix script for doing this version bump and publishing. Better to use that then manually bumping I think.